benchmarks/random.h

```diff
@@ -0,0 +1,20 @@
+// Generating random data
+
+#pragma once
+
+#include <algorithm>
+#include <cstdint>
+#include <functional>
+#include <random>
+#include <vector>
+
+
+::std::vector<::std::uint64_t> GenerateRandom64(::std::size_t count) {
+  ::std::vector<::std::uint64_t> result(count);
+  ::std::random_device random;
+  auto genrand = [&random]() {
+    return random() + (static_cast<::std::uint64_t>(random()) << 32);
+  };
```

Comment:
Is there a reason to do it this way instead of:
```
  std::random_device rd;
  std::default_random_engine gen(rd());
  std::uniform_int_distribution<uint64_t> dist;
  std::generate(result.begin(), result.end(), std::bind(dist, gen));
```